operators/api/v1alpha1/common_types.go

```diff
@@ -0,0 +1,20 @@
+package v1alpha1
+
+// NameCreated contains info about the status of a resource
+type NameCreated struct {
```

Comment:
Not clear to me. Should this be `NamespaceStatus` or is it meant to be more general?